runtime/doc/lua.txt

```diff
@@ -9,24 +9,13 @@ Lua engine                                                           *lua* *Lua*
                                        Type |gO| to see the table of contents.
 
 ==============================================================================
-INTRODUCTION                                                       *lua-intro*
+INTRODUCTION
 
-The Lua 5.1 script engine is builtin and always available. Try this command to
-get an idea of what lurks beneath: >
-
-    :lua print(vim.inspect(package.loaded))
-
-Nvim includes a "standard library" |lua-stdlib| for Lua.  It complements the
```

Comment:
prefer to leave this here. The general ideas here are important enough that we want people to see this when they first go to `:help lua`, without a redirect